.github/workflows/task-unit-test.yml

```diff
@@ -30,7 +30,24 @@ jobs:
       - name: pre-checkout script
         if: ${{ inputs.pre-checkout-script }}
         run: ${{ inputs.pre-checkout-script }}
+      - name: Check for node20 support
+        id: node20
+        run: |
+          for os in amazonlinux:2 ubuntu:bionic; do
```

Comment:
is there a more general way to test support?
something similar to `which node20`?

Reply:
I think this is the simplest way (as we do in RediSearch). The node20 comes from the runner but fails in runtime due to a missing and unsupported glibc version, so I don't think `which node20` will fail/return nothing

Reply:
This is a temporary solution regardless. We hope to drop the support for these OSs soon. When we do, I hope it is clear that this step and all the ones that need its conclusion should be removed